packages/@uppy/utils/src/RateLimitedQueue.js

```diff
@@ -1,3 +1,17 @@
+/**
+ * Array.prototype.findIndex ponyfill for old browsers.
```

Comment:
*polyfill?

Reply:
https://ponyfill.com/ :)

a polyfill should try to follow the spec; this 'ponyfill' only handles the common case and doesn't worry about any edge cases. we know what the arrays it receives look like, so we don't have to worry about holey arrays or what happens if the array changes mid-iteration etc

Reply:
Ah! I didn't know about this. Thank you for clarifying! 👌